diff --git a/config.def.h b/config.def.h
index 86c7447..3cb333d 100644
--- a/config.def.h
+++ b/config.def.h
@@ -145,8 +145,8 @@ static unsigned int cursorshape = 2;
  * Default columns and rows numbers
  */
 
-static unsigned int cols = 80;
-static unsigned int rows = 24;
+static unsigned int cols = 130;
+static unsigned int rows = 35;
 
 /*
  * Default colour and shape of the mouse cursor
@@ -208,7 +208,7 @@ ResourcePref resources[] = {
  * Internal mouse shortcuts.
  * Beware that overloading Button1 will disable the selection.
  */
-const unsigned int mousescrollincrement = 1;
+const unsigned int mousescrollincrement = 4;
 static MouseShortcut mshortcuts[] = {
 	/* mask                 button   function        argument       release */
 	{ XK_ANY_MOD,           Button4, kscrollup,      {.i = 1},      0, /* !alt */ -1 },
