diff --git a/config.def.h b/config.def.h
index fb02872..638a08b 100644
--- a/config.def.h
+++ b/config.def.h
@@ -218,6 +218,7 @@ ResourcePref resources[] = {
  * Internal mouse shortcuts.
  * Beware that overloading Button1 will disable the selection.
  */
+unsigned int mousescrollincrement = 1;
 static MouseShortcut mshortcuts[] = {
 	/* mask                 button   function        argument       release */
 	{ XK_ANY_MOD,           Button4, kscrollup,      {.i = 1},      0, /* !alt */ -1 },
diff --git a/st.c b/st.c
index de58cc1..e16be1b 100644
--- a/st.c
+++ b/st.c
@@ -1093,7 +1093,9 @@ tswapscreen(void)
 void
 kscrolldown(const Arg* a)
 {
-	int n = a->i;
+	int n = (mousescrollincrement > a->i)
+		? mousescrollincrement
+		: a->i;
 
 	if (n < 0)
 		n = term.row + n;
@@ -1111,7 +1113,9 @@ kscrolldown(const Arg* a)
 void
 kscrollup(const Arg* a)
 {
-	int n = a->i;
+	int n = (mousescrollincrement > a->i)
+		? mousescrollincrement
+		: a->i;
 
 	if (n < 0)
 		n = term.row + n;
diff --git a/st.h b/st.h
index e9901d9..95f809a 100644
--- a/st.h
+++ b/st.h
@@ -130,3 +130,4 @@ extern unsigned int tabspaces;
 extern unsigned int defaultfg;
 extern unsigned int defaultbg;
 extern unsigned int defaultcs;
+extern unsigned int mousescrollincrement;
