diff --git a/config.def.h b/config.def.h
index 638a08b..989e7d4 100644
--- a/config.def.h
+++ b/config.def.h
@@ -183,6 +183,7 @@ static uint forcemousemod = ShiftMask;
  */
 ResourcePref resources[] = {
 		{ "font",         STRING,  &font },
+		{ "font2",        STRING,  &font2 },
 		{ "color0",       STRING,  &colorname[0] },
 		{ "color1",       STRING,  &colorname[1] },
 		{ "color2",       STRING,  &colorname[2] },
@@ -204,14 +205,23 @@ ResourcePref resources[] = {
 		{ "cursorColor",  STRING,  &colorname[258] },
 		{ "termname",     STRING,  &termname },
 		{ "shell",        STRING,  &shell },
+		{ "doubleclicktimeout",INTEGER,&doubleclicktimeout },
+		{ "tripleclicktimeout",INTEGER,&tripleclicktimeout },
+		{ "allowaltscreen",INTEGER,&allowaltscreen },
+		{ "allowwindowops",INTEGER,&allowwindowops },
 		{ "minlatency",   INTEGER, &minlatency },
 		{ "maxlatency",   INTEGER, &maxlatency },
 		{ "blinktimeout", INTEGER, &blinktimeout },
+		{ "cursorthickness",INTEGER,&cursorthickness },
+		{ "cursorstyle",  INTEGER, &cursorstyle },
 		{ "bellvolume",   INTEGER, &bellvolume },
 		{ "tabspaces",    INTEGER, &tabspaces },
 		{ "borderpx",     INTEGER, &borderpx },
 		{ "cwscale",      FLOAT,   &cwscale },
 		{ "chscale",      FLOAT,   &chscale },
+		{ "cols",         INTEGER, &cols },
+		{ "rows",         INTEGER, &rows },
+		{ "mousescrollincrement",INTEGER,&mousescrollincrement },
 };
 
 /*
